docs/manage/_reports.rst

```diff
@@ -50,21 +54,26 @@ Open the **Quizzes** sub-tab from the main **Reports** view to access the progre
       .. figure:: img/exam-report-detail.png
         :alt: 
 
-
 #. Click on the attempt icons to see the answers for each one. In the screenshot above learner **Abby L.** had 3 attempts on *Question 8*: the first one was incorrect, the second one (selected) was correct, but the third and finally submitted one was incorrect.
 
-.. _exercise_progress:
+    .. _exercise_progress:
+
+    .. note:: 
+      
+      **Review Exercise Progress**
+
+      When a learner answers exercise questions in Kolibri, the progress bar below the exercise is taking into account only the most recent given answers, meaning that the learner must complete the required number of correct answers (*check marks*) **in the row** for the exercise to be considered completed.
```

Comment:
```suggestion
      As the questions of an exercises are completed, the progress bar below the exercise only takes the most recently given answers into account. This means that the learner must complete the required number of correct answers (*check marks*) **in the row** for the exercise to be considered completed.
```

I simplified the text and added `ly` in `recently`. But I don't actually understand the latter part:

> the progress bar below the exercise only takes the most recently given answers into account

Reply:
Active voice is preferable, as it makes the subject of the sentence clearer and easier to translate, so I would prefer the original formulation instead of your suggestion. 

"the most recent" refers to "in the row" strategy, so _N most recent correct answers_ (_in the row_ is implicit).

Reply:
Okay, will figure something out... it's currently a 302 character sentence, so I wanted to simplify it.

Reply:
:stuck_out_tongue_closed_eyes: 

> Active voice is preferable, as it makes the subject of the sentence clearer and easier to translate

Reply:
Edited!